features/steps/test_starter/{{ cookiecutter.repo_name }}/src/{{ cookiecutter.python_package }}/settings.py

```diff
@@ -25,11 +25,13 @@
 # CONF_SOURCE = "conf"
 
 # Class that manages how configuration is loaded.
-# from kedro.config import TemplatedConfigLoader
-# CONFIG_LOADER_CLASS = TemplatedConfigLoader
+# CONFIG_LOADER_CLASS = ConfigLoader
 # Keyword arguments to pass to the `CONFIG_LOADER_CLASS` constructor.
 # CONFIG_LOADER_ARGS = {
-#     "globals_pattern": "*globals.yml",
+#       "config_patterns": {
+#           "spark" : ["spark*/"],
+#           "parameters": ["params*", "params*/**", "**/params*"],
```

Comment:
What doesn't work exactly? It doesn't override where to fetch parameters from?

Reply:
It does overrides, I just blindly uncomment it and get error because the default are using parameters instead of params, I guess it is actually the purpose.

Reply:
Yeah so the above would work if you're using a naming convention for your parameters with `params` instead of `parameters`. I wanted to illustrate that you can override the default settings. Does that make sense or is it confusing?

Reply:
At the moment the default `settings.py` is a bit messy, I think there is a ticket to clean it up but we never get it into sprint yet. (i.e. ProjectHook should be removed from there)

In general, I think the `settings.py` provide the "default" value as a hint that the user can uncomment and change it.



Examples for this would be `CONF_SOURCE`, `KedroContext` `DataCatalog`, with the exception that we put `TemplateConfigLoader` and `ShelveStore` there.

Personally, I like the idea that the `settings.py` just provides the default and as a hint that this is a knob that I can switch. Uncommenting it will work just fine.

Reply:
I've changed it to be "parameters" again. In the docs https://github.com/kedro-org/kedro/pull/1971 I added more examples and explanation also about the `TemplatedConfigLoader` settings.